File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cc
#include <stdio.h>

int main()
{
    int c, n, i, j, sum, score[1000];
    float avg;
    scanf("%d", &c);

	for (i = 0; i < c; i++) {
        sum = 0;
		scanf("%d", &n);
		for(j = 0; j < n; j++) {
            scanf("%d", &score[j]);
            sum += score[j];
        }
        avg = (float)sum / (float)n;
        sum = 0;
        for(j = 0; j < n; j++) {
            if(avg < score[j]) {
                sum++;
            }
        }
        avg = (float)sum / (float)n * 100;
        printf("%.3f%%\n", avg);
	}
    return 0;
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
c = int(input())

for i in range(c):
    num = list(map(int, input().split()))
    avg = sum(num[1:])/num[0]
    count = 0
    for i in range(1, len(num)):
        if num[i] > avg:
            count += 1
            
    print('%.3f' % (count / num[0] * 100) + '%')